crates/bevy_input/src/lib.rs

```diff
@@ -134,6 +136,7 @@ impl Plugin for InputPlugin {
                 .register_type::<RawGamepadEvent>()
                 .register_type::<RawGamepadAxisChangedEvent>()
                 .register_type::<RawGamepadButtonChangedEvent>()
+                .register_type::<Gamepad>()
```

Comment:
We can also add `GamepadInput`, `GamepadAxis`, and `GamepadButton` now, right? Unless those are registered elsewhere?

Reply:
Added.